### PR TITLE
Better docs

### PR DESCRIPTION
**Pull request by bobturneruk**
Reorganises docs moving content into readthedocs where appropriate leading to a more readable readme.

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## How it works
+
+Image progessing is performed using the 'pygwytracing' script
```
Comment (ns-rse) [inline docs/introduction.md:3]: Typo...

```suggestion
Image processing is performed using the 'pygwytracing' script
```

File: README.md [hunk 2]
```diff
@@ -8,121 +8,21 @@
 
 | [How it works](#how-it-works) | [Run using Docker](#run-using-docker) | [Parameter Configuration](#parameter-configuration) | [Licence](#licence) | [Publications](#publications) | [Contributing](contributing.md) |
```
Comment (ns-rse) [inline README.md:9]:
```suggestion
| [How to use](#How-to-use) | [Installation](#installation) | [Tutorials and Examples](#tutorials-and-Examples) | [Contributing](contributing.md) | [Licence](#licence) | [Citation](#citation) |  |
```

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks for splitting this all up @bobturneruk .

Very minor suggestions, and we'll have to go through the process of documenting with clear examples once we have changes merged to `dev`.

**Comment by bobturneruk** [comment]
Preview docs here https://topostats.readthedocs.io/en/better-docs/

**Review by ns-rse — APPROVED** [review]
The Markdown inline `mark-up` hasn't carried through to the ReadTheDocs HTML for some reason but that is minor and can be corrected in due course.

**Review by SylviaWhittle — APPROVED** [review]
Looks very good - I'll update the 'how it works' to reflect what is done in `topotracing.py` and `grainstats.py` after this pull.

**Pull request merged**